### ISSUE TITLE
Swappable EV Batteries

### ISSUE THREAD

**Issue by lukas-reineke** [ocpi/ocpi#472]
Sorry if this is not the right place to ask.

As far as I can tell, there is currently no support for swappable EV batteries in the spec. Is something like this planned for the future?

**Comment by reinierl** [comment]
No, there is no such support, and no, this is not currently planned for the future.

We'd be happy of course to welcome you to the  [EV Roaming Foundation](https://evroaming.org/) and work with you on a way to support it.

**Comment by mbayings** [comment]
As Reinier mentioned we currently do not have such support. However it is a development that we also see. Issue is a bit that we need input with concrete proposals for solutions. That can only be done via Contributors of the EVRoaming Foundation. We would be happy to work on this together with you.

**Comment by lukas-reineke** [comment]
Thanks for the response.
I've started the process to get us to join the EVRoaming foundation.

Would we need Full Contributor access to work on this with you, or is Associate Contributor enough?

**Comment by lukas-reineke** [comment]
@mbayings sorry for the ping, can you answer my question? Or point me to the right person to ask?
